### PR TITLE
update inspect rules

### PR DESCRIPTION
**Pull request by zhouying7780**
@liangxia Could you please help review and merge, thanks.

### CODE REVIEW

**Comment by zhouying7780** [comment]
@kasturinarra Could you please help review ? thanks.

File: lib/rules/cli/4.1.yaml [hunk 1]
```diff
@@ -1150,11 +1150,13 @@
...
     :whitelist: --whitelist=<value>
 :oadm_inspect:
-  :cmd: oc adm inspect <resource_type> <resource_name>
+  :cmd: oc adm inspect
   :options:
     :A: --all-namespaces=<value>
     :all_namespaces: --all-namespaces=<value>
     :dest_dir: --dest-dir=<value>
+    :resource_type: <value>
+    :resource_name: <value>
```
Comment (liangxia) [inline lib/rules/cli/4.1.yaml:1159]: Sort them in alphanumeric order.

Comment (zhouying7780) [reply lib/rules/cli/4.1.yaml:1158]: @liangxia  Updated , thanks, please help review and merge.

**Pull request merged**